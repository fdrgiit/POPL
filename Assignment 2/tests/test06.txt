expr term factor #
PLUS MINUS MULT COMMA #
expr -> term PLUS expr #
expr -> term MINUS expr #
expr -> term #
PLUS -> MINUS #
factor -> LPAREN expr RPAREN #
factor -> NUM #
factor -> REALNUM # 
hello -> PLUS #
##

