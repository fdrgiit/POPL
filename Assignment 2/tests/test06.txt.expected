ERROR CODE 1
ERROR CODE 2
ERROR CODE 3
ERROR CODE 4
